experiments/sock-shop-promProbe/experiment.yaml

```diff
@@ -49,12 +49,705 @@ spec:
+                      - apiGroups: ["litmuschaos.io"]
+                        resources: ["chaosengines", "chaosexperiments", "chaosresults"]
+                        verbs: ["create", "list", "get", "patch", "update", "delete"]
+                    image: "litmuschaos/go-runner:latest"
+                    imagePullPolicy: Always
+                    args:
+                      - -c
+                      - ./experiments -name pod-cpu-hog
+                    command:
+                      - /bin/bash
+                    env:
+                      - name: TOTAL_CHAOS_DURATION
+                        value: "60"
+
+                      ## Number of CPU cores to stress
+                      - name: CPU_CORES
+                        value: "1"
+
+                      ## LOAD CPU WITH GIVEN PERCENT LOADING FOR THE CPU STRESS WORKERS.
+                      ## 0 IS EFFECTIVELY A SLEEP (NO LOAD) AND 100 IS FULL LOADING
+                      - name: CPU_LOAD
+                        value: "100"
+
+                      ## Percentage of total pods to target
+                      - name: PODS_AFFECTED_PERC
+                        value: ""
+
+                      ## Period to wait before and after injection of chaos in sec
+                      - name: RAMP_TIME
+                        value: ""
+
+                      ## It is used in pumba lib only
+                      - name: LIB_IMAGE
+                        value: "litmuschaos/go-runner:latest"
+
+                      ## It is used in pumba lib only
+                      - name: STRESS_IMAGE
+                        value: "alexeiled/stress-ng:latest-ubuntu"
+
+                      ## provide the cluster runtime
+                      - name: CONTAINER_RUNTIME
+                        value: "containerd"
+
+                      # provide the socket file path
+                      - name: SOCKET_PATH
+                        value: "/run/containerd/containerd.sock"
+
+                      - name: TARGET_CONTAINER
+                        value: ""
+
+                      - name: TARGET_PODS
+                        value: ""
+
+                      - name: DEFAULT_HEALTH_CHECK
+                        value: "false"
+
+                      # To select pods on specific node(s)
+                      - name: NODE_LABEL
+                        value: ""
+
+                      ## it defines the sequence of chaos execution for multiple target pods
+                      ## supported values: serial, parallel
+                      - name: SEQUENCE
+                        value: "parallel"
+
+                    labels:
+                      name: pod-cpu-hog
+                      app.kubernetes.io/part-of: litmus
+                      app.kubernetes.io/component: experiment-job
+                      app.kubernetes.io/runtime-api-usage: "true"
+                      app.kubernetes.io/version: ci
+          - name: pod-delete-rm5
```

Comment:
Will appending experiments to same template works? can you test this once?

Reply:
HCE also follows the same convention, right? Can you describe why this might not work?

Reply:
I am not sure, I asked you if it will work or not.